src/xdocs/checks/javadoc/writetag.xml.template

```diff
@@ -85,15 +85,15 @@
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/Example4.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/Example4.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example4-code">
         Example:
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/Example4.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/Example4.java"/>
```

Comment:
Please create Example5 that has same config as Example4 but `<property name="tagSeverity" value="error"/>`

Reply:
Done